### PR TITLE
Update eval_test.md

### PR DESCRIPTION
**Pull request by chg0901**
由于是第一次做文档迁移翻译，所以比较保守，尽可能的按照原来的comment或者原文翻译，没有添加太多的补充润色

Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmagic/blob/main/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmagic/blob/main/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/migration/eval_test.md [hunk 1]
```diff
@@ -1 +1,159 @@
-# 评测与测试的迁移（待更新）
+# 评估与测试设置的迁移
+
+
+我们更新了 MMagic 1.x 中的评估设置，重要修改如下：
+- 评估字段被分为`val_evaluator`和`test_evaluator`， `interval`被移动到`train_cfg.val_interval`。
```
Comment (RangeKing) [inline docs/zh_cn/migration/eval_test.md:5]:
```suggestion
- 评估字段被分为 `val_evaluator` 和 `test_evaluator`，`interval` 被移动到 `train_cfg.val_interval`。
```

Comment (RangeKing) [reply docs/zh_cn/migration/eval_test.md:5]: 中文 和 英文、数字之间需要加空格，下同

**Review by RangeKing — COMMENTED** [review]
参照 https://mmcv.readthedocs.io/en/latest/community/contributing.html#configure-pre-commit 配置 pre-commit，修复 lint 问题

File: docs/zh_cn/migration/eval_test.md (hunk 1 above)
Comment (chg0901) [reply docs/zh_cn/migration/eval_test.md]: > 

已经通过lint

Comment (chg0901) [reply docs/zh_cn/migration/eval_test.md]: > 中文 和 英文、数字之间需要加空格，下同

修改后，已经通过lint

**Review by zengyh1900 — APPROVED** [review]
lgtm.

**Pull request merged**